chia/full_node/mempool.py

```diff
@@ -122,11 +123,13 @@ def __init__(self, mempool_info: MempoolInfo, fee_estimator: FeeEstimatorInterfa
                 """CREATE TABLE spends(
                 coin_id BLOB NOT NULL,
                 tx BLOB NOT NULL,
-                UNIQUE(coin_id, tx))
+                latest_unspent_id BLOB,
```

Comment:
is there a good reason to not update `coin_id` to always refer to the latest unspent ID instead?

Reply:
is that a problem? as I understand it, the only thing we use this table for is to find spends to evict from the mempool on a new peak. That seems like a pretty reasonable place to hook FF re-validation into.

Reply:
A mempool item can have multiple spends related to one singleton, and just using `coin_id` to keep track of the latest version'd ID didn't seem natural, as that would mean we lose all the original coin IDs that belong to a transaction, and get just a single row with the singleton's latest ID in this table, which is inconsistent with the information on the coins that this item spend, that you find everywhere else (coin spends list, spends list in conditions, bundle coin spends...etc).